weekly-openmeteo/load.py

```diff
@@ -1,15 +1,16 @@
 '''This script loads the formatted sunrise and sunset times onto the postgres database'''
 import logging
+from psycopg2.extras import execute_values
 from dotenv import load_dotenv
 from extract import get_connection
 
 def load_data(data: list[list]) -> None:
     '''Inserts data into a postgres database'''
     query = '''INSERT INTO solar_feature (county_id,sunrise_timestamp,sunset_timestamp)
-    VALUES (%s,%s,%s)'''
+    VALUES %s'''
     with get_connection() as conn:
         with conn.cursor() as cur:
-            cur.executemany(query, data)
+            execute_values(cur,query, data)
```

Comment:
Good switch to make it more efficient with execute values

Reply:
execute_values essentially allows bulk insertion of flattened values, much more efficient that iterative insertion. Important addition for efficiency.